setup/pi/create-backingfiles-partition.sh

```diff
@@ -38,24 +37,24 @@ then
     # Force creation of filesystems even if previous filesystem appears to exist
     mkfs.ext4 -F -L mutable /dev/sda1
     mkfs.xfs -f -m reflink=1 -L backingfiles /dev/sda2
+  fi
     
-    BACKINGFILES_MOUNTPOINT="$1"
-    MUTABLE_MOUNTPOINT="$2"
-    if grep -q backingfiles /etc/fstab
-    then
-      setup_progress "backingfiles already defined in /etc/fstab. Not modifying /etc/fstab."
-    else
-      echo "LABEL=backingfiles $BACKINGFILES_MOUNTPOINT xfs auto,rw,noatime 0 2" >> /etc/fstab
-    fi
-    if grep -q 'mutable' /etc/fstab
-    then
-      setup_progress "mutable already defined in /etc/fstab. Not modifying /etc/fstab."
-    else
-      echo "LABEL=mutable $MUTABLE_MOUNTPOINT ext4 auto,rw 0 2" >> /etc/fstab
-    fi
-    setup_progress "Partitions creation complete."
-    exit 0
+  BACKINGFILES_MOUNTPOINT="$1"
+  MUTABLE_MOUNTPOINT="$2"
+  if grep -q backingfiles /etc/fstab
+  then
+    setup_progress "backingfiles already defined in /etc/fstab. Not modifying /etc/fstab."
+  else
+    echo "LABEL=backingfiles $BACKINGFILES_MOUNTPOINT xfs auto,rw,noatime 0 2" >> /etc/fstab
+  fi
+  if grep -q 'mutable' /etc/fstab
+  then
+    setup_progress "mutable already defined in /etc/fstab. Not modifying /etc/fstab."
+  else
+    echo "LABEL=mutable $MUTABLE_MOUNTPOINT ext4 auto,rw 0 2" >> /etc/fstab
   fi
+  setup_progress "Mount Points updated /etc/fstab."
```

Comment:
it will log this every time, even when it didn't update /etc/fstab

Reply:
What would be a better log message here? ```Partitions setup complete.```  ?

. I want there to be a clear statement that says create-backingfiles-partition  is over.

Reply:
I'm a fan of "Done" myself :)

Reply:
Oh well.. Brevity is not my strong point. Modified it.